scripts/circle-ci-tests.sh

```diff
@@ -57,10 +57,10 @@ else
             echo "Finding PEP 8 violations and storing report..."
             paver run_pep8 > pep8.log || { cat pep8.log; EXIT=1; }
 
-            echo "Finding pylint violations and storing in report..."
+#            echo "Finding pylint violations and storing in report..."
             # HACK: we need to print something to the console, otherwise circleci
             # fails and aborts the job because nothing is displayed for > 10 minutes.
-            paver run_pylint -l $LOWER_PYLINT_THRESHOLD:$UPPER_PYLINT_THRESHOLD | tee pylint.log || EXIT=1
+#            paver run_pylint -l $LOWER_PYLINT_THRESHOLD:$UPPER_PYLINT_THRESHOLD | tee pylint.log || EXIT=1
```

Comment:
@attiyaIshaque Please undo these comments since the build is green now and we don't want to create confusion for others.

Reply:
I will remove this. You can merge this as it's already green. @attiyaIshaque

Reply:
@hasnain-naveed okay.